src/test/regress/bin/normalize.sed

```diff
@@ -92,3 +92,12 @@ s/read_intermediate_result\('insert_select_[0-9]+_/read_intermediate_result('ins
 # ignore job id in repartitioned insert/select
 s/repartitioned_results_[0-9]+/repartitioned_results_xxxxx/g
 
+s/dbname=regression/dbname=<db>/g
+s/\b576[0-9][0-9]\b/xxxxx/g
```

Comment:
We sometimes care if the port is worker 1 port number or worker 2 port number, so replacing all by the same string is not the best we can do.